src/components/organisms/PreviewConfigurationEditor/SortableList/index.ts

```diff
@@ -0,0 +1 @@
+export {default} from './SortableList';
```

Comment:
this was meant to be added in another PR when I finish implementing it, but it doesn't affect anything so I'll leave it here